config/locales/en.yml

```diff
@@ -4,7 +4,7 @@ en:
       frequency:
         daily: You’ll get one email a day
         weekly: You’ll get one email a week
-        immediate: You’ll get an email each time we add or update a page
+        immediately: You’ll get an email each time we add or update a page
```

Comment:
Could we move this to the first commit? It's a bit confusing it's here as the previous key would not actually be found when finding this locale for an immediate email.

Reply:
I'd like to keep it here, as it's the motivation behind the specs in this commit, which will fail without this change. While ideally each commit should be correct in itself, I think it's OK to have other commits to fix things, as long as they tell a clear story. I've updated the commit message to explain better, though.